neuroanalysis/neuronsim/sim.py

```diff
@@ -58,158 +177,131 @@ def all_objects(self):
     def time(self):
         return self._time
 
-    def run(self, samples=1000, **kwds):
+    def run(self, samples:int=1000, **kwds) -> SimState:
```

Comment:
This was the only point of all the changes in this file. I also accidentally hit the "clean up all style issues" button, which I usually try to avoid.